src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -49,8 +50,7 @@ const gridRowDivider = (
 export const AdvancedSearchPage = () => {
   const { t } = useTranslation();
   const history = useHistory();
-  const showFilterDivider = useMediaQuery((theme: Theme) => theme.breakpoints.up('sm'));
-  const isLargeScreen = useMediaQuery((theme: Theme) => theme.breakpoints.up('lg'));
+  const isLargeScreen = useMediaQuery((theme: Theme) => theme.breakpoints.up('sm'));
```

Comment:
Er large screen definert som medium og opp? Eller burde vi kalt denne `isSmallScreen`/`isMobileScreen` og invertert logikken? Eller `isNotSmallScreen`? Egentlig bare pirk som du ikke trenger å gjøre noe med though.

Reply:
Enig i at det er et dårlig navn ja, men syns det så bedre ut når vi baserte alt på bare en her, i stedet for å skifte layout to ganger (både ved lg og sm). Tenkte å bevare `showFilterDivider` eller noe i den duren også, men syns det ble rart siden den også påvirker andre ting 🤔

Reply:
Om vi skulle gjort noe her tenker jeg vi skulle flytta layout conditionals til CSS i stedet for en hook med mange sjekker, men tar ikke det nå. Så tenker det bare får bli `isLargeScreen` selv om det er et dårlig navn jeg altså 🤔